pkg/helpers/helpers.go

```diff
@@ -0,0 +1,34 @@
+package helpers
+
+import (
+	// "fmt"
+	"bytes"
+	"encoding/json"
+)
+
+// JSONMarshalIndent marshals formatted JSON w/ optional SetEscapeHTML
+func JSONMarshalIndent(content interface{}, prefix, indent string, escape bool) ([]byte, error) {
```

Comment:
Thanks!

We could change the name to `JSONMarshalIndentWithEscape`, but kinda long.

The reason I made it configurable is because I wasn't sure if we'd never want to include the standard golang implementation (angle brackets "<" and ">" are escaped to "\u003c" and "\u003e")